icons/list-filter-plus.json

```diff
@@ -0,0 +1,15 @@
+{
+  "$schema": "../icon.schema.json",
+  "contributors": [
+    "abdeniz"
+  ],
+  "tags": [
+    "filter",
+    "plus",
+    "options",
+    "add"
+  ],
+  "categories": [
+    "text"
```

Comment:
Can you also add the `layout` category here.

Reply:
Done ✅